packages/installer/cos.sh

```diff
@@ -40,7 +35,102 @@ fi
 
 ## COMMON
 
+load_full_config() {
```

Comment:
tested this manually to check the order:

```
$ ./cos.sh upgrade
where is verify coming from: config
$ VERIFY=ENV ./cos.sh upgrade
where is verify coming from: ENV
$ VERIFY=ENV ./cos.sh upgrade --no-verify
where is verify coming from: flag
$ ./cos.sh upgrade --no-verify 
where is verify coming from: flag
```

Message output was a temp change to set the proper value on the `_VERIFY` var, print that and exit